### ISSUE TITLE
Should fail for missing frontmatter

### ISSUE THREAD

**Issue by domenic** [smikes/test262-parser#9]
This file

```js
module.exports = function justANormalJSFile() {
    console.log("I'm not a test262 test at all!");
};
```

gives back a dummy parse result, instead of failing.

**Comment by smikes** [comment]
Good point. If there was ever a good reason to accept non-tests, that time is gone and early failure is a better strategy. 

Will get to this tomorrow, currently dealing w/ local power outage :-/

**Comment by smikes** [comment]
I added a property `isATest` to the result.  Is that enough, or do we need an actual throw?

(See version 2.0.6, just published).

**Comment by cscott** [comment]
See issue #10.

**Comment by smikes** [comment]
Property isATest exists since 2.0.6, which was published to npm but not github :-0

Now published both here and on npm as 2.0.7.  Closing this issue - please let me know if more is needed.
